src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import duke.handler.Parser;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    private static String STORAGE_PATH;
```

Comment:
Perhaps this instance variable should be declared final instead? (as your variable name is already in all uppercase letters)